platform/pulp/platform/models/repository.py

```diff
@@ -1,10 +1,268 @@
 """
-Django models related to repositories.
+Repository related Django models.
 """
+from datetime import datetime
 
-from pulp.platform.models import Model
+from django.db import models
+from django.contrib.contenttypes import fields
 
+from pulp.platform.models import Model, Notes, Scratchpad, Config
 
-class Distributor(Model):
+
+class Content(Model):
     # placeholder
-    pass
+    type = models.TextField()
```

Comment:
Class and instance attributes are namespaced to the class (as I'm sure you know) and implicitly apply to the class or instance.  The `type()` builtin function (not a reserved word) unfortunately exists in the global namespace.  I'm opposed to redundantly qualifying it.  Naming it something like `repository_name` does not add clarity and leads to awkward semantics like: `repository.repository_type`.

https://docs.python.org/2/library/functions.html#type